src/main.cpp

```diff
@@ -657,6 +667,8 @@ int main(int Argc, char *Argv[]) {
                 errorHandler.AddError("Unsupported value for --target-os, supported values are: %s",
                                       g->target_registry->getSupportedOSes().c_str());
             }
+        } else if (!strcmp(argv[i], "--disable-vectorcall")) {
```

Comment:
Add a warning that it's ignored if the target OS is not Windows, see similar warnings for ``--dllexport`` around line 930.